public/data/Gajendrakm.json

```diff
@@ -0,0 +1,17 @@
+{
+    "name": "Gajendra KM",
+    "bio": "Full Stack Developer",
+    "avatar": "https://github.com/Gajendrakm.png",
+    "links": [
+      {
+        "name": "Follow me on GitHub",
+        "url": "https://github.com/Gajendrakm",
+        "icon": "github"
+      },
+      {
+        "name": "Follow me on Twitter",
+        "url": "https://twitter.com/tweetgajendra",
+        "icon": "twitter"
+      }
+    ]
+  }
```

Comment:
GitHub is highlighting that this needs an extra blank line - we can merge and this can be fixed later

Reply:
@eddiejaoude should every file made, have an empty line at the end? thanks.